src/components/atoms/menuItem/MenuItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./MenuItem.module.scss";
+
+interface MenuItemProps {
+  name: string;
+  ingredients: string[];
+  price: number;
+}
+
+const MenuItem: React.FC<MenuItemProps> = ({ name, ingredients, price }) => {
+  return (
+    <div className={styles["menu-item"]}>
+      <div className={styles["menu-item__info-box"]}>
+        <h4 className={styles["menu-item__info-box__title"]}>{name}</h4>
+        <p className={styles["menu-item__info-box__description"]}>
```

Comment:
What do you think if we change the description to ingredients? Because we style ingredients taken from props.
I think it is more readable.

Reply:
> What do you think if we change the description to ingredients? Because we style ingredients taken from props. I think it is more readable.

great! i will change it